src/steamship/cli/create_instance.py

```diff
@@ -0,0 +1,119 @@
+import json
+from os import path
+from typing import Any, Dict, Optional
+
+import click
+
+from steamship import Steamship, SteamshipError
+from steamship.cli.ship_spinner import ship_spinner
+from steamship.data.manifest import DeployableType, Manifest
+
+
+@click.command(name="use")
+@click.option(
+    "--workspace",
+    "-w",
+    required=False,
+    type=str,
+    help="Workspace handle. The new instance will be created in this workspace. If not specified, "
+    "the default workspace will be used.",
+)
+@click.option(
+    "--instance",
+    "-i",
+    type=str,
+    required=False,
+    help="Instance handle. Used to name the instance (if it needs to be created). If not specified, "
+    "a generated instance name will be used.",
+)
+@click.option(
+    "--config",
+    "-c",
+    type=str,
+    required=False,
+    help="Instance configuration. May be inline JSON or a path to a file. If not specified, "
+    "an empty configuration dictionary will be passed to the instance.",
+)
+def create_instance(
+    workspace: Optional[str] = None, instance: Optional[str] = None, config: Optional[str] = None
+):
+    """Create an instance of your package/plugin for use.
+
+    Must be run from a directory containing a Steamship manifest."""
+    global client
+    global manifest
+
+    try:
+        client = Steamship(workspace=workspace)
+    except SteamshipError as e:
+        click.secho(e.message, fg="red")
+        click.get_current_context().abort()
+
+    if path.exists("steamship.json"):
+        manifest = Manifest.load_manifest()
+    else:
+        click.secho("No manifest found for instance creation.", fg="red")
+        click.secho("Please try again from a directory with a package or plugin manifest.")
+        click.get_current_context().abort()
+
+    if manifest is None:
+        click.secho("Steamship manifest failed to load.", fg="red")
+        click.get_current_context().abort()
+
+    invocable_config = config_dict(config)
+
+    instance_fn = client.use
+    if manifest.type == DeployableType.PLUGIN:
+        instance_fn = client.use_plugin
+
+    try:
+        click.echo("Creating a new instance for usage: ", nl=False)
+        with ship_spinner():
+            instance = instance_fn(
+                manifest.handle,
+                instance_handle=instance,
```

Comment:
I think this might be an issue, or at least unexpected - 

If you pass the same instance handle twice, or don't pass it twice, this will fail, right?  because the version won't match the second time? 

If I'm right, I would be in favor of allowing the user to easily confirm deletion of the old one, or maybe it's a command line flag, or maybe that's the default behavior.

Reply:
there are any number of cases where the instance collisions could be an issue, i suppose. you could imagine "random" workspaces for each invocation dealing with that. but we could also add a flag. do you have a pref?